src/main/resources/view/CustomerListCard.fxml

```diff
@@ -31,6 +31,7 @@
       <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
       <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
       <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
+      <Label fx:id="commissions" styleClass="cell_small_label" text="\$commissions" />
```

Comment:
This `\$commissions` text is still present when the commissions are shown below, will need to remove in the future.